apps/minifront/src/components/tx-details/tx-viewer.tsx

```diff
@@ -10,19 +10,28 @@ import { useQuery } from '@tanstack/react-query';
 import fetchReceiverView from './hooks';
 import { classifyTransaction } from '@penumbra-zone/perspective/transaction/classify';
 import { uint8ArrayToHex } from '@penumbra-zone/types/hex';
+import { viewClient } from '../../clients';
+import { ChainRegistryClient } from '@penumbra-labs/registry';
 
 export enum TxDetailsTab {
   PUBLIC = 'public',
   PRIVATE = 'private',
-  RECIEVER = 'reciever',
+  RECEIVER = 'receiver',
 }
 
 const OPTIONS = [
   { label: 'Your View', value: TxDetailsTab.PRIVATE },
   { label: 'Public View', value: TxDetailsTab.PUBLIC },
-  { label: 'Reciever View', value: TxDetailsTab.RECIEVER },
+  { label: 'Receiver View', value: TxDetailsTab.RECEIVER },
 ];
 
+const getMetadata: MetadataFetchFn = async ({ assetId }) => {
+  const feeAssetId = assetId ? assetId : new ChainRegistryClient().bundled.globals().stakingAssetId;
```

Comment:
Given `stakingAssetId` should never change, in this PR I often grab this from the bundle